pipeline/load.py

```diff
@@ -24,45 +22,44 @@ def get_db_connection():
         return {'error': 'Unable to connect to the database.'}
 
 
-def split_df(df: pd.DataFrame) -> tuple:
-    pass
-
-
 def upload_stories(df: pd.DataFrame) -> None:
```

Comment:
Might be worth changing function name to something more general like 'load' as its uploading more than just stories